dask/dask.yaml

```diff
@@ -43,6 +43,9 @@ optimization:
     subgraphs: null  # true for dask.dataframe, false for everything else
     rename-keys: true
 
+delayed:
+  fuse: false
```

Comment:
mixed feelings about the config name. There is a dedicated `optimization` section and the other collections don't have any optimization controlling options.

Reply:
putting it into the fuse section with

```
optimisations:
   fuse:
       delayed
```

messes with the optimiser :(

Reply:
what? how? Well... never mind then 💢

Reply:
I **think** this was caused by a typo on my end :(

Changed things to put delayed under fuse